app/views/spotlight/sir_trevor/blocks/_iframe_block.html.erb

```diff
@@ -0,0 +1 @@
+<%= sanitize iframe_block.code, tags: %w(iframe), attributes: %w(id width height src frameborder allowfullscreen sandbox seamless srcdoc) %>
```

Comment:
Yeah, I only took the HTML5 list and validated it against a couple `<iframe>` providers. I'll add them if you think we should.